ui/pages/scalable-rtr-react/.yarnrc.yml

```diff
@@ -0,0 +1 @@
+nodeLinker: node-modules
```

Comment:
Not sure if the two files above should be checked in. If not, we should ignore them.

Reply:
probably not, we should ignore.